tests/remote/test_devedition.py

```diff
@@ -39,18 +39,18 @@ def test_release_scraper(tmpdir, args, url):
 
 @pytest.mark.ci_only
 @pytest.mark.parametrize("args,url", [
-    ({'application': 'devedition', 'platform': 'linux', 'version': '60.0b1', 'build_number': 1},
```

Comment:
Every test case here has build 3 in its URL, for example:

> devedition/candidates/60.0b1-candidates/**build3**/linux-i686/en-US/firefox-60.0b1.tar.bz2

Before the fix, if build with requested build_number was not found, then first build was used. So these tests started failing after the fix because build_number 1 was used as an argument.

So I have fixed tests to use build_number 3.

Reply:
Oh! What an oversight when we created those tests. Good that we cover those cases now. Thanks for explaining.